css/style.css

```diff
@@ -0,0 +1,27 @@
+
+html{
+
+    box-sizing: border-box;
+}
+
+*, *::before, *::after{
+
+    box-sizing: inherit;
+}
+
+body { 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after "{" of a multi-line block (block-opening-brace-newline-after)](https://app.codacy.com/gh/WebHeroSchool/miniProject/pullRequest?prid=5576209)